### PR TITLE
add classes Main and A by Stelmashok

### PR DESCRIPTION
**Pull request by ViktorMarchuk**
ok

### CODE REVIEW

File: A.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+public class A {
+    @Override
+    public String toString() {
+        return "Wow it's a kind of magic! Awesome! Keep it on ;)";
+    }
```
Comment (ViktorMarchuk) [inline A.java:5]: class A - ok

Comment (dmitriystel) [reply A.java:5]: thanks

Comment (ViktorMarchuk) [reply A.java:5]: ok,  i am merge

File: Main.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+public class Main {
+    public static void main(String[] args) {
+        A a = new A();
+        System.out.println(a);
+    }
```
Comment (ViktorMarchuk) [inline Main.java:5]: add system.out

Comment (dmitriystel) [reply Main.java:5]: done

**Pull request merged**